File: Java/Program/Clothing.java
public class Clothing extends Product{
    String size;
    String material;
    String gender;

    public Clothing(){
        
    }

    //Getter dan setter untuk size
    public String getsize(){
        return this.size;
    }
    public void setsize(String size){
        this.size = size;
    }

    //Getter dan setter untuk material
    public String getmaterial(){
        return this.material;
    }

    public void setmaterial(String material){
        this.material = material;
    }

    //Getter dan setter untuk gender
    public String getgender(){
        return this.gender;
    }

    public void setgender(String gender){
        this.gender = gender;
    }
}

File: CPP/Program/Clothing.cpp
#include <iostream> 
#include <string>
#include "Product.cpp"
using namespace std;

class Clothing : public Product {
private:
    string size;
    string material;
    string gender;

public:
    Clothing(){

    }

    // Getter dan setter untuk size
    string getSize() {
        return this->size;
    }

    void setSize(string size) {
        this->size = size;
    }

    // Getter dan setter untuk material
    string getMaterial() {
        return this->material;
    }

    void setMaterial(string material) {
        this->material = material;
    }

    // Getter dan setter untuk gender
    string getGender() {
        return this->gender;
    }

    void setGender(string gender) {
        this->gender = gender;
    }
    ~Clothing(){

    }
};
